src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,35 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int weight;
+    private String fuel;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, int weight, String fuel) {
+        this.weight = weight;
+        this.fuel = fuel;
```

Comment:
In your code **name** and **color** are ignored.
Same for other classes.
```suggestion
    public Bulldozer(String name, String color, int weight, String fuel) {
        super(name, color);
        this.weight = weight;
        this.fuel = fuel;
```